phpcs.security.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Security Standards">
```

Comment:
I wonder if it would be helpful to name this in a way to remind us that this is both Security and Syntax (if I'm reading the notes correctly on what is going on here)?

Reply:
This file is just the security side of that check. The syntax part of that check is performed by the other line in the `composer test` script - https://github.com/MITLibraries/mitlib-wp-network/pull/20/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R154

Reply:
Ah got it thanks. So _that's_ what that gibberish line of code does lol.